tensorflow/python/ops/image_ops_test.py

```diff
@@ -4073,6 +4073,14 @@ def testShape(self):
         self.assertEqual(image.get_shape().as_list(),
                          [None, None, channels or None])
 
+  def testEmptyTensor(self):
+    with self.cached_session(use_gpu=True) as sess:
+      image = array_ops.reshape(
+          constant_op.constant([], dtypes.uint8), [0, 0, 3])
+      with self.assertRaisesOpError(r"image data should not be empty"):
+        png = image_ops.encode_png(image)
+
```

Comment:
`File "/third_party/py/absl/third_party/unittest3_backport/case.py", line 37, in testPartExecutor
    yield
  File "third_party/py/absl/third_party/unittest3_backport/case.py", line 162, in run
    testMethod()
  File "third_party/tensorflow/python/ops/image_ops_test.py", line 4082, in testEmptyTensor
    png = image_ops.encode_png(image)
  File "<embedded stdlib>/contextlib.py", line 24, in __exit__
    self.gen.next()
  File "third_party/tensorflow/python/framework/test_util.py", line 2808, in assertRaisesWithPredicateMatch
    (str(type(e)), str(e)))
AssertionError: Exception of type <type 'exceptions.AssertionError'>: OpError not raised`

Reply:
@yongtang can you please fix above error ?